tests/integration/cmd_dev_test.go

```diff
@@ -936,6 +937,111 @@ ComponentSettings:
 		})
 	}
 
+	for _, podman := range []bool{true, false} {
+		podman := podman
+		Context(fmt.Sprintf("multiple dev sessions with different project are running on same platform (podman=%v), same port", podman), helper.LabelPodmanIf(podman, func() {
+			if runtime.GOOS == "darwin" {
+				Skip("cannot run this test out of the box on macOS because the test uses a custom address in the range 127.0.0/8 and for macOS we need to ensure the addresses are open for request before using them; Ref: https://superuser.com/questions/458875/how-do-you-get-loopback-addresses-other-than-127-0-0-1-to-work-on-os-x#458877")
```

Comment:
I think `Skip` needs to be used on Ginkgo subject/setup nodes (`BeforeEach` for example). It cannot be used on `Context`: https://onsi.github.io/ginkgo/#skipping-specs